commonmark/pom.xml

```diff
@@ -42,6 +43,19 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
```

Comment:
What does this do, is it needed?

Reply:
This plugin updates the MANIFEST.MF file with osgi metadata.

Reply:
I meant what does the `<extensions>true</extensions>` do?

Reply:
Yes it is needed. Please see docs: https://maven.apache.org/guides/mini/guide-using-extensions.html .

That is how the plugin configuration is done for it to work.